subprojects/language-scala/src/integTest/groovy/org/gradle/language/scala/UpToDatePlatformScalaCompileIntegrationTest.groovy

```diff
@@ -31,6 +32,7 @@ class UpToDatePlatformScalaCompileIntegrationTest extends AbstractIntegrationSpe
         file('app/controller/Person.scala') << "class Person(name: String)"
     }
 
+    @Ignore("TODO upgrade Scala to be Java 9+ compatible or make workers honor javaHome to enable this test again")
```

Comment:
@oehme What is the problem here? Scala works on Java 9, only incremental scala compilation does not work (i.e. the compilation is always non-incremental). So running this test with Java 8/9 should be fine, right?

Reply:
Let me try that again, maybe it was due to some other problem that has now been fixed.

Reply:
Nope, still failing: https://builds.gradle.org/viewLog.html?buildId=15657513&tab=buildResultsDiv&buildTypeId=Gradle_Check_Quick_Java10_Oracle_Linux_scala

Is there a specific Scala version that I should be avoiding?

Reply:
It's the scala version after all: https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html

This test uses 2.11.8 currently